roles/openshift_gcp/tasks/main.yml

```diff
@@ -1,43 +1,53 @@
 #
-# This role relies on gcloud invoked via templated bash in order to
-# provide a high performance deployment option. The next logical step
-# is to transition to a deployment manager template which is then instantiated.
 # TODO: use a formal set of role parameters consistent with openshift_aws
 #
 ---
-- name: Templatize DNS script
-  template: src=dns.j2.sh dest=/tmp/openshift_gcp_provision_dns.sh mode=u+rx
-- name: Templatize provision script
-  template: src=provision.j2.sh dest=/tmp/openshift_gcp_provision.sh mode=u+rx
-- name: Templatize de-provision script
-  template: src=remove.j2.sh dest=/tmp/openshift_gcp_provision_remove.sh mode=u+rx
-  when:
-  - state | default('present') == 'absent'
-
-- name: Provision GCP DNS domain
-  command: /tmp/openshift_gcp_provision_dns.sh
-  args:
-    chdir: "{{ files_dir }}"
-  register: dns_provision
-  when:
-  - state | default('present') == 'present'
-
-- name: Ensure that DNS resolves to the hosted zone
```

Comment:
Is this lost in your change?  This is the most valuable part for a user.

Reply:
No, it was just moved to the `dns-zone.yml` file.